lib/db/schema/patch-091-090.sql

```diff
@@ -10,7 +10,7 @@
 -- DROP PROCEDURE updateTotpToken_3;
 -- DROP PROCEDURE resetAccount_10;
 -- DROP PROCEDURE sessionWithDevice_17;
--- DROP PROCEDURE accountRecord_5;
+-- DROP PROCEDURE accountRecord_4;
```

Comment:
This was a typo I noticed along the way, migration 91 introduced the `accountRecord_4` SP, there is no `accountRecord_5` yet.